resources/providers/aws_cis/logging/provider.go

```diff
@@ -0,0 +1,86 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package logging
+
+import (
+	"context"
+	s3Types "github.com/aws/aws-sdk-go-v2/service/s3/types"
+	"github.com/elastic/cloudbeat/resources/fetching"
+	"github.com/elastic/cloudbeat/resources/providers/awslib"
+	"github.com/elastic/cloudbeat/resources/providers/awslib/cloudtrail"
+	"github.com/elastic/cloudbeat/resources/providers/awslib/s3"
+)
+
+type EnrichedTrail struct {
+	cloudtrail.TrailInfo
+	BucketInfo TrailBucket `json:"bucket_info"`
+}
+
+type TrailBucket struct {
+	Grants  []s3Types.Grant `json:"grants,omitempty"`
+	Policy  s3.BucketPolicy `json:"policy,omitempty"`
+	Logging s3.Logging      `json:"logging,omitempty"`
+}
+
+func (p *Provider) DescribeTrails(ctx context.Context) ([]awslib.AwsResource, error) {
+	trails, trailsErr := p.trailProvider.DescribeTrails(ctx)
```

Comment:
For all the errors in this flow
```suggestion
	trails, err := p.trailProvider.DescribeTrails(ctx)
```